Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM gentoo/stage3-amd64
 ---> f99615487d17
Step 2/2 : RUN emerge --sync  2>&1 &&  ACCEPT_KEYWORDS="~amd64" emerge   net-misc/curl sys-process/vixie-cron net-misc/socat  2>&1
 ---> Using cache
 ---> a22d6744c9d3
Successfully built a22d6744c9d3
Successfully tagged mygentoo-stage3-amd64:latest
Fri, 11 Jan 2019 05:06:05 UTC
Testing branch: master
Progress: 0/1/21
==Running le_test_dependencies please wait
no crontab for root
no crontab for root
curl installed. [PASS]
crontab installed. [PASS]
openssl installed. [PASS]
socat installed. [PASS]
------------------------------------------
Progress: 0/2/21
==Running le_test_install please wait
./acme.sh install [PASS]
/root/.acme.sh/acme.sh exists. [PASS]
_contains '7 0 * * * "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" > /dev/null' '0 \* \* \* "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" > /dev/null' [PASS]
/root/.acme.sh/acme.sh uninstall  > /dev/null [PASS]
------------------------------------------
Progress: 0/3/21
==Running le_test_uninstall please wait
./acme.sh install [PASS]
/root/.acme.sh/acme.sh uninstall [PASS]
/root/.acme.sh/acme.sh no exists. [PASS]
equals  [PASS]
------------------------------------------
Progress: 0/4/21
==Running le_test_installtodir please wait
./acme.sh install [PASS]
/root/myle/acme.sh exists. [PASS]
_contains '8 0 * * * "/root/myle"/acme.sh --cron --home "/root/myle" > /dev/null' '0 \* \* \* "/root/myle"/acme.sh --cron --home "/root/myle" > /dev/null' [PASS]
/root/myle/acme.sh uninstall [PASS]
------------------------------------------
Progress: 0/5/21
==Running le_test_uninstalltodir please wait
./acme.sh install [PASS]
/root/myle/acme.sh uninstall [PASS]
/root/myle/acme.sh no exists. [PASS]
equals  [PASS]
------------------------------------------
Progress: 0/6/21
==Running le_test_install_config_home please wait
./acme.sh --install --config-home /root/etc/acme [PASS]
/root/.acme.sh/acme.sh exists. [PASS]
_contains '9 0 * * * "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" --config-home "/root/etc/acme" > /dev/null' '0 \* \* \* "/root/.acme.sh"/acme.sh --cron --home "/root/.acme.sh" --config-home "/root/etc/acme" > /dev/null' [PASS]
/root/etc/acme/account.conf exists. [PASS]
/root/.acme.sh/account.conf no exists. [PASS]
/root/.acme.sh/acme.sh --cron --config-home /root/etc/acme > /dev/null [PASS]
/root/.acme.sh/acme.sh --uninstall --config-home /root/etc/acme > /dev/null [PASS]
/root/etc/acme/account.conf exists. [PASS]
/root/.acme.sh/account.conf no exists. [PASS]
------------------------------------------
Progress: 0/7/21
==Running le_test_standandalone_renew please wait
/root/.acme.sh/acme.sh issue no testdocker.acme.sh [FAIL] 
Run Failed
deactivate failed
[Fri Jan 11 05:06:12 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:12 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:13 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:13 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
cat: /root/.acme.sh/ca/acme-staging.api.letsencrypt.org/ca.conf: No such file or directory
[Fri Jan 11 05:06:13 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:13 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 1/8/21
==Running le_test_standandalone_renew_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --certpath '/acmetest/certs/domain.cer' --keypath '/acmetest/certs/domain.key'  --capath '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchainpath  '/acmetest/certs/full.cer' [FAIL] 
Run Failed
deactivate failed
[Fri Jan 11 05:06:15 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:16 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:17 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:17 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:17 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:17 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 2/9/21
==Running le_test_standandalone_renew_localaddress_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --local-address 0.0.0.0 --certpath '/acmetest/certs/domain.cer' --keypath '/acmetest/certs/domain.key'  --capath '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchainpath  '/acmetest/certs/full.cer' [FAIL] 
Run Failed
deactivate failed
[Fri Jan 11 05:06:19 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:19 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:21 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:21 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:21 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:21 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 3/10/21
==Running le_test_standandalone_listen_v4_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --listen-v4 --cert-file '/acmetest/certs/domain.cer' --key-file '/acmetest/certs/domain.key'  --ca-file '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchain-file  '/acmetest/certs/full.cer' [FAIL] 
Run Failed
deactivate failed
[Fri Jan 11 05:06:23 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:23 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:24 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:24 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:24 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:24 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 4/11/21
==Running le_test_standandalone_listen_v6_v2 please wait
Skipped by TEST_IPV6
deactivate failed
[Fri Jan 11 05:06:25 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:25 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:26 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:26 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:26 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:26 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 5/12/21
==Running le_test_standandalone_deactivate_v2 please wait
/root/.acme.sh/acme.sh --issue -d testdocker.acme.sh --standalone --certpath '/acmetest/certs/domain.cer' --keypath '/acmetest/certs/domain.key'  --capath '/acmetest/certs/ca.cer'  --reloadcmd 'echo this is reload'  --fullchainpath  '/acmetest/certs/full.cer' [FAIL] 
Run Failed
deactivate failed
[Fri Jan 11 05:06:28 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:29 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:30 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:30 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:30 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:30 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 6/13/21
==Running le_test_standandalone please wait
/root/.acme.sh/acme.sh issue no testdocker.acme.sh [FAIL] 
Run Failed
deactivate failed
[Fri Jan 11 05:06:32 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:32 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:33 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:33 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:33 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:33 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 7/14/21
==Running le_test_standandalone_SAN please wait
/root/.acme.sh/acme.sh issue no "testdocker.acme.sh" "testdocker2.acme.sh" [FAIL] 
Run Failed
deactivate failed
[Fri Jan 11 05:06:35 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:36 -00 2019] Getting new-authz for domain='testdocker.acme.shtestdocker2.acme.sh'
[Fri Jan 11 05:06:37 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:37 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:37 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:37 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 8/15/21
==Running le_test_standandalone_ECDSA_256 please wait
Skipped by NO_ECC_CASES
deactivate failed
[Fri Jan 11 05:06:37 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:38 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:39 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:39 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:39 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:39 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 9/16/21
==Running le_test_standandalone_ECDSA_256_renew please wait
Skipped by NO_ECC_CASES
deactivate failed
[Fri Jan 11 05:06:39 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:40 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:41 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:41 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:41 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:41 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 10/17/21
==Running le_test_standandalone_ECDSA_256_SAN_renew please wait
Skipped by NO_ECC_CASES
deactivate failed
[Fri Jan 11 05:06:41 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:42 -00 2019] Getting new-authz for domain='testdocker.acme.shtestdocker2.acme.sh'
[Fri Jan 11 05:06:43 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:43 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:43 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:43 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 11/18/21
==Running le_test_standandalone_ECDSA_256_SAN_renew_v2 please wait
Skipped by NO_ECC_CASES
deactivate failed
[Fri Jan 11 05:06:43 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:44 -00 2019] Getting new-authz for domain='testdocker.acme.shtestdocker2.acme.sh'
[Fri Jan 11 05:06:45 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:45 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:45 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:45 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 12/19/21
==Running le_test_standandalone_ECDSA_384 please wait
Skipped by NO_ECC_CASES
deactivate failed
[Fri Jan 11 05:06:45 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:46 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:47 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:47 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:47 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:47 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 13/20/21
==Running le_test_standandalone_renew_idn_v2 please wait
Skipped by TEST_IDN
deactivate failed
[Fri Jan 11 05:06:47 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:47 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:49 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:49 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:49 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:49 -00 2019] Can not get domain new authz token.
------------------------------------------
Progress: 14/21/21
==Running le_test_dnsapi please wait
Skipped by TEST_DNS
deactivate failed
[Fri Jan 11 05:06:49 -00 2019] Using stage ACME_DIRECTORY: https://acme-staging.api.letsencrypt.org/directory
[Fri Jan 11 05:06:49 -00 2019] Getting new-authz for domain='testdocker.acme.sh'
[Fri Jan 11 05:06:51 -00 2019] It seems there is an error, but it's recovered now, please try again.
[Fri Jan 11 05:06:51 -00 2019] If you see this message for a second time, please report bug: https://github.com/Neilpang/acme.sh
[Fri Jan 11 05:06:51 -00 2019] new-authz error: {"type":"urn:acme:error:unauthorized","detail":"No registration exists matching provided key","status": 403}
[Fri Jan 11 05:06:51 -00 2019] Can not get domain new authz token.
------------------------------------------
